### PR TITLE
RDAMT-231: Updated GitIgnore to ignore private configuration files, 

### PR DESCRIPTION
**Pull request by peterchafe**
and moved base64 to separate classes.

### CODE REVIEW

File: DotNet/.gitignore [hunk 1]
```diff
@@ -37,4 +37,5 @@ _ReSharper*/
 packages/
 output/
 .jsbeautifyrc
-*.orig
\ No newline at end of file
+*.orig
+CompanyLogo.cs
```
Comment (stephenmatis) [inline DotNet/.gitignore]: How does this exclude auth_config.json from Git?

**Review by stephenmatis — COMMENTED** [review]
Settings.cs line 9 has an error with CompanyLogo. Did you forget to add the file?

File: DotNet/.gitignore (hunk 1 above)
Comment (peterchafe) [reply DotNet/.gitignore]: Hmm, that is weird, I accidentally added it to the local git ignored files. Didn't know that was a thing!

Comment (peterchafe) [reply DotNet/.gitignore]: I have made the necessary change to the  web/.gitignore, and it is now working as required.

**Comment by stephenmatis** [comment]
I'm confused about CompanyLogo.cs_Sample. Are we supposed to edit the filename each time we do development, then change it back to _Sample before committing? I still cannot get your branch to build.

**Comment by peterchafe** [comment]
@stephenmatis, I have pushed the latest changes. could you please review?

Thank you.

**Review by stephenmatis — APPROVED** [review]

**Pull request merged**